EEBOS/owl-ui
# Bug/sc 1127 type declarations

### Short summary

- `d.ts` files were not being created correctly
- css wasn't being minified correctly
- storybook wasn't generating css correctly

---

### Test steps

1. purge node modules
2. yarn install
3. yarn start:storybook
4. yarn start:web

storybook and web should load css correctly

`*.d.ts` in `dist` folders should contain types, interfaces, etc.
